voluncheer/voluncheer/settings.py

```diff
@@ -97,26 +101,20 @@
 WSGI_APPLICATION = "voluncheer.wsgi.application"
 ASGI_APPLICATION = "voluncheer.asgi.application"
 
-REDIS_CHATROOM_PORT = os.getenv("REDIS_CHATROOM_PORT")
 if environment.is_aws:
-    CHANNEL_LAYERS = {
-        "default": {
-            "BACKEND": "django_redis.cache.RedisCache",
-            "LOCATION": REDIS_CHATROOM_PORT,
-            "OPTIONS": {
-                "CLIENT_CLASS": "django_redis.client.DefaultClient",
-            },
-        }
-    }
+    REDIS_CHATROOM_PORT = os.getenv("REDIS_CHATROOM_PORT")
```

Comment:
```suggestion
    REDIS_CHATROOM_PORT = os.getenv("REDIS_CHATROOM_PORT", "127.0.0.1")
```

Then you can remove the branching logic.